internal/schema/resources_state.go

```diff
@@ -0,0 +1,6 @@
+package schema
+
+// ResourcesState is the state of a list of resources.
+type ResourcesState struct {
```

Comment:
Should this match the struct name in `output` (`ResourceState`)?

Also, I think this should have fields for `TotalHourlyCost` and `TotalMonthlyCost`, etc.

Reply:
Yes, that's the output representation of this struct.

I was thinking about that, since we used to calculate the hourly and monthly costs on the output phase I didn't do it but if you agree I think it could be a field in this struct and be filled after the cost calculation phase and before generating the output.

Reply:
Ah right, yeah make sense. I think your suggestion of doing it at the calculation phase is good.